src/preamble.js

```diff
@@ -930,11 +930,11 @@ function stackTrace() {
 
 // Memory management
 
-var PAGE_SIZE = 4096;
+var PAGE_SIZE = 65536; // Wasm pages are fixed to 64k.
```

Comment:
There is certainly some code somewhere that assumes memory pages will be 4k, because it's 4k on pretty much all popular platforms (x86, ARM, Mips, PPC(?) etc). But for wasm it will be impractical to actually implement 4k granularity on top of 16k underlying pages. So given that I think it would be best to go to 16k everywhere. Not sure about filesystem page size though. Are you talking about mmap-type mapping of files, or is there something relevant to filesystems that's not just a mirror of the memory page size?